src/finnish-fish-soup.md

```diff
@@ -0,0 +1,37 @@
+# Finnish fish soup
+
+Traditional Finnish fish soup usually served with rye bread.
+Originally from a finnish book _Saiturin Keittiö_ (The Cheapskates Kitchen) from 1992. 
+
+- ⏲️ Time: 50 min
+- 🍽️ Servings: 4
+- 💲  Cost ~8€
+
+## Ingredients
+
+- 600g of small fish (eg. perch) or fish offal (heads, fins, ...)
+- 6 midsized potatoes
+- small packet of frozen spinach or 50g of parboiled fresh spinach
+- 1 tbsp of wheat flour
+- 2 egg yellows
+- 1/2 dl of cream
```

Comment:
Can't say I've seen a `dl` in a cooking recipe before. Is this a `decilitre` or something else? Might be best to use more standard measurement units.

Reply:
dl does indeed stand for decilitre. I think it's the norm in europe. Happy to add another measurement unit if you deem it necessary.

Reply:
Maybe leave it for now, and we can see what Luke's opinion is. All I know is that in Canada it is not common outside of science class :)

Reply:
`dl` is very much used indeed, but if it is rather unknown over the pond then we should add an alternative in parentheses, though I do not myself know what that unit would be.

Reply:
In the states you would say "1/4 cup" (~20% more volume) or "3 tbsp" (~10% less volume). "50mL" is probably a better internationally recognizable measure, most Americans will not have heard the word "deciliter" since high school but our measuring cups have secondary milliliter legends.

Reply:
@hallzy dL (decilitres) are very commonly used in Europe and probably elsewhere. [Deci](https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes) comes from Latin *decimus*, which means tenth. It's also where the word *decimation* comes from; cutting down something by one-tenth. Americans are probably not going to be aware of this, so I would also recommend "50 mL" if it poses a problem to reading comprehension by American readers.

Reply:
@Holsterbau I am aware of what a dL is, i've just never seen it on a recipe before.

Reply:
@hallzy Right. Regardless, I agree with your point. Litres and millilitres are going to be much more obvious to our non-metric friends.